src/WikiService.Data/ArticleRepository.cs

```diff
@@ -12,5 +16,23 @@ public class ArticleRepository : IArticleRepository
     {
       _provider = provider;
     }
+
+    public async Task<bool> DoesSameArticleNameExistAsync(Guid rubricId, string articleName)
```

Comment:
you in article repo, no need use the word "Article" => DoesSameNameExistAsync

Reply:
done